types/wordpress__components/index.d.ts

```diff
@@ -1,8 +1,9 @@
-// Type definitions for @wordpress/components 9.0
+// Type definitions for @wordpress/components 10.0.0
```

Comment:
The linter is complaining about the patch version:

```suggestion
// Type definitions for @wordpress/components 10.0
```

It looks [`9.8`](https://unpkg.com/browse/@wordpress/components@9.8.0/src/flex/) may have been the earliest version to include this:

```suggestion
// Type definitions for @wordpress/components 9.8
```